docs/NMOS With JPEG XS.md

```diff
@@ -46,24 +71,46 @@ Sender resources provide no indication of media type or format, since this is de
+
+- [Bit Rate](https://specs.amwa.tv/nmos-parameter-registers/branches/main/sender-attributes/#bit-rate)  
+  If the Sender complies with ST 2110-22, the Sender resource MUST indicate the bit rate (kilobits/second) including the RTP transport overhead.
+  Otherwise, it is only RECOMMENDED that this attribute is included.
+  The `bit_rate` integer value is expressed in units of 1000 bits per second, rounding up.
+  The value is for the IP packets, including the RTP, UDP and IP packet headers and the payload.
+- [ST 2110-21 Sender Type](https://specs.amwa.tv/nmos-parameter-registers/branches/main/sender-attributes/#st-2110-21-sender-type)  
+  If the Sender complies with the traffic shaping and delivery timing requirements for ST 2110-22, it MUST include the `st2110_21_sender_type` attribute.
+
+An example Sender resource is provided in the [Examples](../examples/).
+
+## JPEG XS IS-04 Receivers
+
+Nodes capable of receiving JPEG XS video streams MUST have a Receiver resource in the IS-04 Node API, which lists `video/jxsv` in the `media_types` array within the `caps` object.
 This has been permitted since IS-04 v1.1.
 
-Nodes implementing IS-04 v1.3 or higher MUST indicate the color (sub-)sampling in the Flow resource using the `components` attribute defined in the [Flow Attributes register](https://specs.amwa.tv/nmos-parameter-registers/branches/main/flow-attributes/) of the NMOS Parameter Registers.
-The `components` array value corresponds to the `sampling`, `width` and `height` values in the SDP format-specific parameters defined by RFC 9134.
+Nodes implementing [BCP-004-01][] Receiver Capabilities use the existing `constraint_sets` parameter within the `caps` object, describing combinations of frame rates, width and height, and other parameters which the receiver can support, using the parameter constraints defined in the [Capabilities register](https://specs.amwa.tv/nmos-parameter-registers/branches/main/capabilities/) of the NMOS Parameter Registers.
 
-Nodes implementing IS-04 v1.3 or higher MUST indicate the stream bit rate in the Flow resource using the `bit_rate` attribute also defined in the Flow Attributes register.
-The bit rate value also appears in the SDP file, per RFC 9134.
+Receivers are RECOMMENDED to use the following parameter constraints:
 
-An example Flow resource is provided in the [Examples](../examples/).
+- [Profile](https://specs.amwa.tv/nmos-parameter-registers/branches/main/capabilities/#profile)
+- [Level](https://specs.amwa.tv/nmos-parameter-registers/branches/main/capabilities/#level)
+- [Sublevel Bits Per Pixel](https://specs.amwa.tv/nmos-parameter-registers/branches/main/capabilities/#sublevel-bits-per-pixel)  
+  When the JPEG XS decoder supports the Full (or Unrestricted) sublevel, the Receiver MAY indicate that this parameter is unconstrained, as per BCP-004-01.
```

Comment:
Could the Full value be specified as the sum of the bit_depth of all the components? This would be the true Full value and should be allowed. Then unconstrained corresponds to not having or having an empty sublevel_bpp constraint.

Reply:
👍 Let's address this alongside whether precise encoder target bpp is allowed.

Reply:
In XS standard, `Full` translates to bpc * num_components (bpc is bits-per-component, which is signaled in the stream and is the same value for all components). So, yes, `Full` equals what you explained.